stempy/image.cpp

```diff
@@ -269,6 +270,62 @@ vector<STEMImage> createSTEMImages(InputIt first, InputIt last,
   return images;
 }
 
+// returns the binding result
+std::vector<double> getContainer(const STEMImage& inImage, const int numBins)
+{
+  // information about input STEMImage
+  int width = inImage.width;
+  int height = inImage.height;
+  auto curData = inImage.data;
+  std::cout << "Input STEM Image has width = " << width
+            << ", height = " << height << std::endl;
+
+  auto result = std::minmax_element(curData.get(), curData.get() + width * height);
+  double min = *result.first;
+  double max = *result.second;
+
+  // the "length" of each slot of the container
+  double length = static_cast<double>((max - min) / numBins);
+
+  std::vector<double> container;
+  // push all the intermediate values
+  for (int i = 0; i <= numBins; i++) {
+    container.push_back(min + i * length);
```

Comment:
I see here, I missed this where you use `<=` numBins, so you should cover the complete range as you have the extra bound at the top.